conda_build/__init__.py

```diff
@@ -15,3 +16,16 @@
     "render",
     "skeleton",
 ]
+
+# Skip context logic for docs since we don't install all dependencies
```

Comment:
```suggestion
# Skip context logic for doc generation since we don't install all dependencies in the CI doc build environment,
# see .readthedocs.yml file
```